kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/utils/ResourceFilter.java

```diff
@@ -0,0 +1,12 @@
+package org.arquillian.cube.kubernetes.impl.utils;
+
+import java.nio.file.Path;
+
+public class ResourceFilter {
+
+    public static boolean filterKubernetesResource(Path p) {
+        return p.toString().toLowerCase().endsWith(".yaml")
+            || p.toString().toLowerCase().endsWith(".yml")
+            || p.toString().toLowerCase().endsWith(".json");
```

Comment:
maybe creating private method and using same for all of three will follow DRY

Reply:
yes we could do but I am not sure it is any real advantage in this specific case